src/Tools/dotnet-monitor/DotNetHost.Unix.cs

```diff
@@ -0,0 +1,64 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using System;
+using System.IO;
+using System.Runtime.InteropServices;
+
+namespace Microsoft.Diagnostics.Tools.Monitor
+{
+    partial class DotNetHost
+    {
+        private sealed class UnixDotNetHostHelper : IDotNetHostHelper
+        {
+            private const string DefaultPathLinux = $"/usr/share/{InstallationDirectoryName}";
```

Comment:
Is this unniversal across all the dotnet supported distros?

Reply:
Yes, it's the final fallback; see https://github.com/dotnet/runtime/blob/main/src/native/corehost/hostmisc/pal.unix.cpp#L596